packages/itwinui-css/src/tabs/tabs.scss

```diff
@@ -99,6 +99,24 @@ $borderless-horizontal-tab-min-height: calc(
     }
   }
 
+  &[data-iui-overflow='true'] {
+    .iui-tab {
+      white-space: nowrap;
+    }
+
+    &.iui-borderless {
```

Comment:
Remove custom scrollbar styling. We should keep it consistent across all components.

Reply:
If I do that, it covers up the highlighted line.

Reply:
That's ok because of this [comment](https://github.com/iTwin/iTwinUI/pull/1060#discussion_r1188446917)